griptape/utils/chat.py

```diff
@@ -16,22 +17,39 @@ class Chat:
     prompt_prefix: str = field(default="Q: ", kw_only=True)
     response_prefix: str = field(default="A: ", kw_only=True)
     output_fn: Callable[[str], None] = field(
-        default=Factory(lambda: print), kw_only=True
+        default=lambda x: print(x, end=""), kw_only=True
     )
+    stream: bool = field(default=False, kw_only=True)
+
+    def start(self, initial_turn: str = None) -> None:
+        if initial_turn:
+            self.output_fn(initial_turn + "\n")
+
+        self.structure.stream = self.stream
+        self.structure.prompt_driver.stream = self.stream
+        if self.stream:
+            self.structure = Stream(self.structure)
```

Comment:
Let's not re-assign `self.structure`. This will change its' type.

Reply:
Will do.